policy/modules/services/radius.te

```diff
@@ -82,7 +82,7 @@ corenet_tcp_connect_snmp_port(radiusd_t)
 
 corenet_sendrecv_generic_server_packets(radiusd_t)
 corenet_udp_bind_generic_port(radiusd_t)
-corenet_dontaudit_udp_bind_all_ports(radiusd_t)
+corenet_udp_bind_all_unreserved_ports(radiusd_t)
```

Comment:
Can you clarify how radiusd is selecting which port to bind to? It seems that we would still want the dontaudit to cover cases where it chooses a reserved port.

Reply:
The default ports of radius are 1812 and 1813 [1]. Add corenet_udp_bind_all_unreserved_ports to make sure it can bind these ports. We can keep the line corenet_dontaudit_udp_bind_all_ports.

[1] https://en.wikipedia.org/wiki/RADIUS

Reply:
In that case, why not have labeling to on those two ports and then only allow binding on them?

Reply:
Thanks, This is because the port 18120 is enabled in configure file. It works when I bind  this port with semanage port.